src/jwplayer/players/DesktopArticleVideoPlayer/DesktopArticleVideoPlayer.module.scss

```diff
@@ -20,10 +20,12 @@
 
 .desktopArticleVideoWrapperScrollPlayer {
 	height: max-content;
-	width: 300px;
 	position: fixed;
 	right: 18px;
 	bottom: 45px;
+	&--customized {
```

Comment:
nitpicking: it's BEM, isn't it? As the concept is nice I'd consider resigning from it here and just going with a `camelCase` convention. This way you'd keep it consistent in the line no. 90 in `DesktopArticleVideoPlayer.tsx`.

Reply:
It is, but you're right; gonna change it.